src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -852,6 +852,11 @@ public static void createAllDirectories()
                 throw new ConfigurationException("saved_caches_directory must be specified", false);
 
             FileUtils.createDirectory(conf.saved_caches_directory);
+
+            if (conf.lock_keyspaces_directory == null)
```

Comment:
should we make this more generic? do we anticipate need to store any other files here?

Reply:
yeah I was also thinking this, probably makes sense to make this a bit more generic, just in-case we need to use this for something else in the future. maybe we call it 'persistent_settings_directory'?

Reply:
👍